src/components/Dashboard/Form/Team/TeamMembers/TeamMembers.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import DocumentForm from "../../DocumentForm";
+import FieldWrapper from "../../FieldWrapper/FieldWrapper";
+import { FieldArray } from "react-final-form-arrays";
+import styles from "./TeamMembers.scss";
+
+const TeamMembers = () => (
+  <DocumentForm.Step>
+    <FieldArray name="members">
+      {
+        ({ fields }) => (
+            <React.Fragment>
+              <button 
+                onClick={() => {fields.push({name: "", function: ""})}}>
+                  Dodaj
+              </button>
+              {
+                fields.map((element, index) => 
```

Comment:
for now it's ok to have index as a key, however docs says: 
>  Only do this if items have no stable IDs

and if we gonna store somewhere data about members (and I guess we will) so they will have unique ID that part should be changed in the future

Also @magul said that the first member will be always team-leader "function: "przewodniczący zespołu"" so we need add this exception

Reply:
Yes, this is only a temporary solution until DB won't be connected. Every detail bounded with the future of this component will be planed later.